samples/apps/bookinfo/route-rule-ratings-test-v2.yaml

```diff
@@ -0,0 +1,17 @@
+type: route-rule
+spec:
+  name: ratings-test-v2
+  destination: ratings.default.svc.cluster.local
+  precedence: 2
+  route:
+  - tags:
+      version: v2
+---
+type: route-rule
+spec:
+  name: reviews-test-v2
```

Comment:
right.. But to keep in line with the tutorials and story flow, I suggest picking up from last step of request routing task (all traffic goes to reviews v3). You wont even need a rule here for reviews